hack/jenkins/job-configs/test-linkchecker.yaml

```diff
@@ -0,0 +1,66 @@
+- job-template:
+    name: 'kubernetes-test-linkchecker'
+    description: |
+        Grab the latest from GitHub, then run hack/verify-linkcheck.sh.<br>
+        Test Owner: Build Cop
+    logrotate:
+        numToKeep: 200
+    builders:
+        - shell: './hack/verify-linkcheck.sh'
+    publishers:
+        - claim-build
+        - gcs-uploader
+        - log-parser
+        - email-ext
```

Comment:
add underneath
```
     recipients: 'you@google.com'
```
or whatever.

(see kubernetes-verify-jenkins-jobs.yaml for an example)